package.json

```diff
@@ -42,28 +55,48 @@
     "single-spa": ">=4.0.0"
   },
   "devDependencies": {
+    "@angular-builders/custom-webpack": "~9.1.0",
     "@angular-devkit/architect": "~0.901.0",
     "@angular-devkit/build-angular": "~0.901.0",
-    "@angular-devkit/core": "^9.1.0",
+    "@angular-devkit/core": "~9.1.0",
     "@angular-devkit/schematics": "^9.1.0",
-    "@angular/compiler": "^9.1.0",
-    "@angular/compiler-cli": "^9.1.0",
-    "@angular/core": "^9.1.0",
-    "@schematics/angular": "^9.1.0",
+    "@angular/animations": "~9.1.0",
+    "@angular/cli": "~9.1.0",
+    "@angular/common": "~9.1.0",
+    "@angular/compiler": "~9.1.0",
+    "@angular/compiler-cli": "~9.1.0",
+    "@angular/core": "~9.1.0",
+    "@angular/forms": "~9.1.0",
+    "@angular/platform-browser": "~9.1.0",
+    "@angular/platform-browser-dynamic": "~9.1.0",
+    "@angular/router": "~9.1.0",
+    "@schematics/angular": "~9.1.0",
     "@types/jest": "^24.0.11",
     "@types/node": "^11.11.3",
     "@types/webpack-merge": "^4.1.3",
     "@typescript-eslint/eslint-plugin": "^1.4.2",
     "@typescript-eslint/parser": "^1.4.2",
     "cpx": "^1.5.0",
+    "cypress": "^4.3.0",
     "eslint": "^5.15.2",
     "jest": "^24.5.0",
     "rimraf": "^2.6.2",
+    "rxjs": "~6.5.5",
+    "start-server-and-test": "^1.10.11",
     "ts-jest": "^24.0.0",
+    "tslib": "^1.11.1",
     "typescript": "3.7.5",
-    "webpack": "^4.29.6"
+    "webpack": "^4.29.6",
```

Comment:
Why is webpack/webpack-cli in this package.json? Shouldn't it only be in the integration/portal package.json?

Reply:
Sure, will move it.